src/mlpack/methods/emd/emd_first_imf.hpp

```diff
@@ -0,0 +1,146 @@
+// emd_first_imf.hpp
```

Comment:
All the files we add should have license information + details at the top (take a look at any other file to see what I mean); do you think you can add those?